pydarn/plotting/fan.py

```diff
@@ -455,21 +441,24 @@ def plot_fan(cls, dmap_data: List[dict], ax=None, ranges=None,
             ticks = locator.tick_values(vmin=zmin, vmax=zmax)
 
             if zmin == 0:
-                cb = ax.figure.colorbar(mappable, ax=ax, extend='max',
-                                        ticks=ticks)
+                extend = 'max'
             else:
-                cb = ax.figure.colorbar(mappable, ax=ax, extend='both',
-                                        ticks=ticks)
+                extend = 'both'
+
+            if cax is None:
+                cax = ax.inset_axes([1.04, 0.0, 0.05, 1.0])
```

Comment:
This overlaps the '06' label on the plot :(

Reply:
Oh that's an easy fix, just have to bump the `1.04` value to something larger.